### ISSUE TITLE
Add option to disable enumeration checking

### ISSUE THREAD

**Issue by eigenfoo** [viresh-ratnakar/exet#25]
Whenever editing a clue, Exet always ensures that the enumeration is "correct", which is usually helpful but may be limiting, especially for variety cryptics. `DUMB LUCK` always has an enum of `(4,4)` and can never be `(8, 2 wds)` or even something funkier.

It would be nice to have an option under "preferences" (off by default) that disables this checking, and allows the user the freedom to write whatever enumeration they want.

**Comment by viresh-ratnakar** [comment]
Sure, will do, soon.

**Comment by viresh-ratnakar** [comment]
I have addressed this with v0.72. But there are a couple of things to note:

- "... (8, 2 wds)" is now parsed as an enum that specifies 8 letters (earlier it was parsed as an enum with an unknown number of letters).
- If an enum specifies a length and it matches the light length, Exet has no complaints. So, in the case listed in the bug, assuming the light also has 8 letters, it will just work now without doing anything special.
- If an enum does not specify a length -- e.g., "... (2 words)" or "... (one word)", then too, now, Exet does not complain.
- If an enum specifies a length that is different from the light length, Exet *does* revert the clue enum to its previous value. I've added a way to prevent that. But it's not a common use case, so I have chosen not to clutter the UI with it. Exolve already has an option called `ignore-enum-mismatch` for exactly this, and I have now made Exet respect it too.
- It bothered me that the way to handle this is a bit obscure. I *have* documented the method in README.md, but I also wanted a way to point the user to it proactively, when relevant.
- So, I've also added a "Tips" section with a few useful tips (that can be cycled through or random-jumped-through). One of the tips is a detailed description of how to add `exolve-option: ignore-enum-mismatch`.
- Here's the satisfying part: if a user-typed enum is reverted because of a mismatch, the relevant Tip (about `ignore-enum-mismatch`) is *automatically surfaced*. I hope to do more of this "contextually relevant surfacing" of useful tips in the future.
